src/spec/methods.cr

```diff
@@ -28,14 +28,15 @@ module Spec::Methods
     end
   end
 
-  def pending(description, file = __FILE__, line = __LINE__, end_line = __END_LINE__, &block)
+  def pending(description = "assert", file = __FILE__, line = __LINE__, end_line = __END_LINE__, &block)
     return unless Spec.matches?(description, file, line, end_line)
 
     Spec.formatters.each(&.before_example(description))
 
     Spec::RootContext.report(:pending, description, file, line)
   end
 
+  # @deprecated Please use {#it} instead
```

Comment:
You can use `DEPRECATED` special markdown tag so the generated code will receive the proper visual handling.

Also use backtick to refer to another method like `it`.

See the documentation section for more details on the recommended doc style:

https://crystal-lang.org/docs/conventions/documenting_code.html

Cheers.